### PR TITLE
fix: offline homepage error

### PR DESCRIPTION
**Pull request by sirugh**
## Description

Network errors are always returned by `useQuery`, even if we use `cache-and-network`. Going offline and then navigating to a cached cms page results in the `data` from the query returning, but the network call being made. Since we're offline, a network error is returned by the hook, which was causing the `Page Fetch Error` to be rendered in the UI.

To be honest the error was really a bad UX anyways. We have better methods for handling page level errors such as toasts.

## Related Issue

Closes [PWA-708](https://jira.corp.magento.com/browse/PWA-708).

### Verification Steps

1. Open PR/deployed home page.
2. Now got to category or products page.
3. Turn network to offline in devtools.
4. click Venia logo to go back to home page.
5. Wait for 3-5 secs.

## Checklist

- [ ] I have added tests to cover my changes, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

**Review by revanth0212 — APPROVED** [review]
Any tests that need to be updated?

File: packages/peregrine/lib/talons/Cms/useCmsPage.js [hunk 1]
```diff
@@ -60,7 +61,6 @@ export const useCmsPage = props => {
     return {
         cmsPage,
         hasContent,
-        error,
```
Comment (jimbo) [inline packages/peregrine/lib/talons/Cms/useCmsPage.js:63]: I think I'd prefer to keep returning this value from the error, even if we don't use it in the component.

**Review by jimbo — COMMENTED** [review]
Makes sense. One change noted.

**Review by jimbo — APPROVED** [review]

File: packages/peregrine/lib/talons/Cms/useCmsPage.js (hunk 1 above)
Comment (sirugh) [reply packages/peregrine/lib/talons/Cms/useCmsPage.js]: Reverted

**Comment by dpatil-magento** [comment]
QA Approved.

**Pull request merged**